Added docs for get orgs command in kubectl-gs (#1544)
giantswarm/docs: src/content/ui-api/kubectl-gs/get-organizations.md

---
linkTitle: get organizations
title: "'kubectl gs get organizations' command reference"
description: Reference documentation on how to list organizations and get details for a single organization using 'kubectl gs'.
weight: 50
menu:
  main:
    parent: uiapi-kubectlgs
aliases:

>   - /reference/kubectl-gs/get-organizations/

Reviewer: aliases are only needed if the page has been moved, to have redirects from old URLs to the new URL. Not needed here.

Author: Removed the alias

owner:
  - https://github.com/orgs/giantswarm/teams/team-rainbow
user_questions:

>   - How can I list organizations in a cluster using kubectl?

Reviewer: ```suggestion
  - How can I list organizations in the management API using kubectl?
```
